.changeset/chatty-ravens-collect.md

```diff
@@ -0,0 +1,29 @@
+---
+"@vivjs/constants": minor
+"@vivjs/extensions": minor
+"@vivjs/layers": minor
+"@vivjs/loaders": minor
+"@hms-dbmi/viv": minor
+"@vivjs/types": minor
+"@vivjs/viewers": minor
+"@vivjs/views": minor
+---
+
+This release potentially includes backward-incompatible changes. To avoid automatically adopting such releases, ensure you are either pinning the exact version of @hms-dbmi/viv in your package.json file or using a version range syntax that only accepts patch updates, such as ~0.16.1. Refer to npm's semver documentation for details.
```

Comment:
```suggestion
**This release includes backward-incompatible changes**. To avoid automatically adopting such releases, ensure you are either pinning the exact version of `@hms-dbmi/viv` in your `package.json` file or using a version range syntax that only accepts patch updates, such as `~0.16.1`. Refer to npm's [semver documentation](https://docs.npmjs.com/cli/v6/using-npm/semver/) for details.
```

Just adding some highlighting. Also I removed "potentially", we know it does.